cmd/sysdump.go

```diff
@@ -26,4 +26,9 @@ var sysdumpCmd = &cobra.Command{
 func init() {
 	rootCmd.AddCommand(sysdumpCmd)
 	sysdumpCmd.Flags().StringVarP(&dumpOptions.Filename, "file", "f", "", "output file to use")
+	sysdumpCmd.Flags().StringVarP(&dumpOptions.Namespace, "namespace", "n", "kubearmor", "Namespace for resources")
```

Comment:
I think there should be no reason to add this `namespace` option. `karmor probe` autodetects the namespace afaik.

Reply:
Ok gonna remove it!